packages/client/shared/gqlIds/JiraServerIssueId.ts

```diff
@@ -1,8 +1,8 @@
 const JiraServerIssueId = {
-  join: (providerId: number, integrationRepoId: string, issueId: string) => `${providerId}:${integrationRepoId}:${issueId}`,
```

Comment:
+1 why the name change out of interest? JiraServerIssue description refers to it as `integrationRepoId`: https://github.com/ParabolInc/parabol/blob/a0951776dcf99f90578746d2c30f3563b3e4e3d3/packages/server/graphql/types/JiraServerIssue.ts#L19

Reply:
@nickoferrall I renamed it to avoid confusion as we have `integrationRepoId` which is `service:providerId:repositoryId`. Renamed to `repositoryId` and fixed the description